README.md

```diff
@@ -330,6 +331,65 @@ end
 
 5. Your app is now able to access the authenticated `ProductsController` from the `HomeController` using session tokens.
 
+### Marking shops as uninstalled on app/uninstalled webhook
+
+To ensure OAuth still works with session tokens, your app must update its shop records in the event a shop uninstalls your app. You can be notified of these events by subscribing to the `app/uninstall` [webhook](https://shopify.dev/docs/admin-api/rest/reference/events/webhook).
+
+#### Setup app/uninstall webhook
+1. Use the `add_webhook` generator from ShopifyApp to setup the `app/uninstalled` webhook for the app.
+
+```shell script
+rails g shopify_app:add_webhook -t app/uninstalled -a {your_app_url}/webhooks/app_uninstalled
+```
+
+This will add the `app/uninstalled` webhook to your config and create the `AppUninstalledJob` job class for you.  
```

Comment:
The suggestion below might not be capturing the correct detail. 
I am hoping we more specifically address "which config is getting updated", "where will the job be created", and "what will that let me do" for clarity.

```suggestion
This will add the `app/uninstalled` webhook to your app config and create the `AppUninstalledJob` job class for you to add uninstallation handler logic.  
```